hack/update-godep-licenses.sh

```diff
@@ -141,21 +145,22 @@ for PACKAGE in $(cat Godeps/Godeps.json | \
   echo "= ${DEPS_DIR}/${PACKAGE} licensed under: ="
   echo
 
-  content=""
+  file=""
   if [[ -n "${CONTENT[${PACKAGE}-LICENSE]-}" ]]; then
-    content="${CONTENT[${PACKAGE}-LICENSE]-}"
+      file="${CONTENT[${PACKAGE}-LICENSE]-}"
   elif [[ -n "${CONTENT[${PACKAGE}-COPYRIGHT]-}" ]]; then
-    content="${CONTENT[${PACKAGE}-COPYRIGHT]-}"
+      file="${CONTENT[${PACKAGE}-COPYRIGHT]-}"
   fi
-  if [[ -z "${content}" ]]; then
-      echo "UNKNOWN"
-      content="/dev/null"
-  else
-      cat "${content}"
+  if [[ -z "${file}" ]]; then
+      echo "Could not find a license for ${PACKAGE} - aborting" > /dev/stderr
```

Comment:
godep could not locate a license/copyright for ${PACKAGE}.  

Options:
1. Contact the author of the package and ensure there is a license and godep can include it correctly
2. Do not submit this package to the Kubernetes tree.